docker/prod.dockerfile

```diff
@@ -26,3 +17,5 @@ ENV JAR="iudx.catalogue.server-cluster-${VERSION}-fat.jar"
 WORKDIR /usr/share/app
 COPY docs docs
 COPY --from=builder /usr/share/app/target/${JAR} ./fatjar.jar
+RUN groupadd -r myuser && useradd -r -g myuser myuser
```

Comment:
This will create a user with uid 999, which is systemd user in debian image, need to create a user who does not `belong default users of ubuntu i.e. create a user with random uid, like a user with uid 1001 as done similarly in bitnami

Reply:
After this change it needs to be tested.